test/integration/consul-container/libs/utils/helpers.go

```diff
@@ -18,3 +27,102 @@ func ApplyDefaultProxySettings(c *api.Client) (bool, error) {
 	ok, _, err := c.ConfigEntries().Set(req, &api.WriteOptions{})
 	return ok, err
 }
+
+// Generates a private and public key pair that is for signing
+// JWT
+func GenerateKey() (pub, priv string, err error) {
```

Comment:
I'm curious about the choice for using named returned parameters here. The code below doesn't really seem structured for them since we are wrapping the errors and explicitly returning constant values, not the named params.

Reply:
I followed some existing pattern from oidcauth code. also we use both pub and private key in the jwt_auth_test.go by doing: `libutils.GenerateKey()` on line 114